specification/containerservice/resource-manager/readme.md

```diff
@@ -692,6 +692,18 @@ input-file:
 - Microsoft.ContainerService/stable/2016-03-30/containerService.json
 ```
 
+### Tag: profile-hybrid-2020-09-01
+
+These settings apply only when `--tag=profile-hybrid-2020-09-01` is specified on the command line.
+Creating this tag to pick proper resources from the hybrid profile.
+
+``` yaml $(tag) == 'profile-hybrid-2020-09-01'
+input-file:
+- Microsoft.ContainerService/stable/2020-11-01/managedClusters.json
+- Microsoft.ContainerService/stable/2019-04-01/location.json
+- Microsoft.ContainerService/stable/2017-07-01/containerService.json
+```
```

Comment:
Will this affect other languages? Do they expect same?

Not sure about `containerService.json`, this is likely deprecated?

Reply:
This one isn't an override but is a creation of the base directly from the versions that are provided in the profile version JSON. As is stands before, this tag just didn't exist for the other languages so it's not something that could affect anything that was already existing.

Reply:
Got it. The other question is whether `containerService.json` is required here? I think current Java mgmt does not include it.

Reply:
From what I see everything in there is deprecated but I don't know myself why this was included in the hybrid profile. Everything in the package-2020-09 tag and below includes `2020-09-01/managedClusters.json` with the `containerService.json` but was removed after that. Apart from Java though it is a snapshot in time when this deprecated API can be used? I didn't seem to have any conflicts during Java codegen from this.

Reply:
we have not shipped containerservice SDK for any other language. Java is the first. We should be good here